### DISCUSSION TITLE
target 82 gets no 100% match width box-shadows

### DISCUSSION THREAD

**Discussion by verberg** [cssbattle/forum#204] [Q&A]
I have a solution width 100% match but 238 chars. That's why im looking vor an other way. I found one, but I can't get 100%. 
Any other width this problem? I tried about more than 50 different margins!

<style>*>*{width:30;height:30;margin:135 183;color:#F3AC3C;transform:rotate(45deg);box-shadow:0 0 0 35px,-65px 0,0 -65px,0 0 0 65px#998235,0 0 0 9in,inset 9in 0#1A4341

**Comment by SelenIT** [comment]
Apparently thin greenish lines near the "cutouts" in the outer "border" prevent this kind of solutions from getting 💯. You can make the "patches" a bit larger to cover these lines: `-65px 0 0 .9px,0 -65px 0 .9px` instead of just `-65px 0,0 -65px` seem to work.

**Reply by verberg** [reply]
Yes it works! Thank's a lot (I'm now on the global rank 20 - I never thought I could reach this...)
But just tell me: How you got this idea? I would never get this idea? - I try on chrome...

**Reply by gunnarbittersmann** [reply]
I also got my first 💯 matches with tiny spread radii. Some iterations later I realized that I don’t need them anymore.

**Reply by SelenIT** [reply]
My first 💯 was with two shadows for each “patch”, one slightly shifted inside and another slightly shifted outside. I tried maybe several hundreds variations of single shadows with no spread, but haven’t got any luck so far 🤪

**Reply by gunnarbittersmann** [reply]
> But just tell me: How you got this idea?

When you look at the rendered result, you see that the yellow box shadows don’t exactly cover the whole width of the green diamond, hence they need to be a tiny bit bigger.

<img width="400" alt="Bildschirmfoto 2021-12-28 um 09 23 41" src="https://user-images.githubusercontent.com/409844/147545894-894e15e6-08e5-43b4-9b08-3946ea4b678e.png">

Turns out there’s not much range for what “a tiny bit” is. I had tried `1q` and `.9q` with no luck. As @SelenIT found out it has to be `.9px` in your case.

**Reply by rafehimself** [reply]
q works but its `1.2q` so overall 2 bigger with a spread are longer than 4 shifted in my case, ` 0 1.2q,  0 1.2q` additional 14 chars

> I realized that I don’t need them anymore.

most likely a secret for a top2

**Reply by gunnarbittersmann** [reply]
I tell you a secret: when the yellow should cover the green, one way is to make the yellow bigger… 😏

**Reply by rafehimself** [reply]
yes, I realized it 😃 🥈  but now it's easier for others as well 😝 
in other words, perfection do not always win on CSSBattle 🤣

**Reply by gunnarbittersmann** [reply]
On many targets, I try my best to be as imperfect as possible. But that’s another secret… 😏

**Reply by gunnarbittersmann** [reply]
You begin to realize that the “buggy” checking algorithm is your friend, not your enemy? 😆

**Reply by rafehimself** [reply]
nah, I would call that "the dark side" of CSSBattle, but if you can't get rid of the shadow, u gotta learn to live with it ☯️
